packages/api/src/router/comment.ts

```diff
@@ -0,0 +1,9 @@
+import { createTRPCRouter, protectedProcedure } from "../trpc";
+
+export const commentRouter = createTRPCRouter({
+  getComments: protectedProcedure.query(async ({ ctx }) => {
+    return await ctx.db.comment.findMany({
+      orderBy: { createdAt: "desc" },
+    });
+  }),
+});
```

Comment:
instead of creating a new endpoint just for comments, can you just add the comments to the getPosts endpoint?